calculation/sha_calc/sha_calc/directivity/bea20/bench_data/create_bench.py

```diff
@@ -28,7 +28,7 @@ def create_benchmark_data():
         x, y = np.meshgrid(lon_values, lat_values)
         site_coords = np.stack((x, y), axis=2).reshape(-1, 2)
 
-        fd, _, _, _, _, _ = directivity.compute_directivity_hypo_averaging(
+        fd, _ = directivity.compute_directivity_srf_multi(
```

Comment:
The function `compute_directivity_srf_multi()` returns `compute_fault_directivity()`
and it returns two different formats?
within `compute_fault_directivity()` 
```python
if hyp_down_dip == 1 and hyp_along_strike == 1:
    return _compute_directivity_effect()
    # and this _compute_directivity_effect returns the following variables
    # fd, (phi_red, predictor_functions, other)
else:
    return fdi_average, fdi_array, phired_average
```
It means there is a change that we could face `not enough values to unpack (expected 3, got 2) ?
Would it be safer to return `fdi_average, (fdi_array, phired_average)` for else case?

Reply:
Could actually get rid of the entire if statement so now all returns should be the same with 3 returns
`if hyp_down_dip == 1 and hyp_along_strike == 1:`
Good spotting